resources/eventing/README.md

```diff
@@ -0,0 +1,58 @@
+# Eventing Chart
+
+This helm charts contains all components required for eventing in Kyma:
+
+Components:
+- event-publisher-proxy
+
+## Event-Publisher-Proxy
+
+This component receives Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects it to the Enterprise Messaging Service Cloud Event Gateway. See [here](https://github.com/kyma-project/kyma/tree/master/components/event-publisher-proxy) for more details.
+
+## Install
+
+You can install this helm chart either using Helm or via Kyma CLI. In both cases, the secret details for BEB have to be configured using the `beb` prefixed variables.
```

Comment:
I'm not sure if the command below (`$ kyma install -s <source-image> -o installation-overrides-epp.yaml`) refers to `the secret details for BEB have to be configured using the beb prefixed variables`. If not, instead of a `:` we should describe what the command does.